Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gravitl/go-builder as builder
+FROM gravitl/go-builder AS builder
```

Comment:
is this change intended?

Reply:
Yes, in the new docker version,   the syntax  check becomes more strict.   "FROM,  as"  receives warning message.   "FROM,  AS" works with both of UPPERCASE.